app/views/layouts/_download_link.html.erb

```diff
@@ -1,5 +1,5 @@
 <% if @user_embargoed %>
-    <span class="label label-warning embargoed-file"><i class="fas fa-ban"></i> Not available until <%= @study.embargo.to_s(:long) %></span>
+    <span class="label label-warning embargoed-file"><i class="fas fa-ban"></i> Data embargo expires on <%= @study.embargo.to_s(:long) %></span>
```

Comment:
"expires on" still seems ambiguous to me. Would "Last date of data embargo" be any better?

(My pedantic spouse says "Expires <date> <time>" I suggested 12:01 am for <time>, he says 00:01 but lack of units is also odd). In any event, I'll go with whatever seems clear to most. My brain is now too alert to the issue to be useful. Sorry!